### PR TITLE
EQ-387: Add ability to load schemas from s3

### PR DESCRIPTION
**Pull request by dhilton**
### Changes

This PR enables the survey runner to load schemas from s3. A new environment variable is
added called EQ_SCHEMA_BUCKET which if set, tries to load schemas from an s3 bucket, using
IAM permissions when deployed to Elasticbeanstalk or picks up standard AWS boto credential
set.
### How to test
1. Create a new s3 bucket and upload a json schema to it, ensuring that the schema is not named the same as any existing schemas.
2. Set the environment variable `EQ_SCHEMA_BUCKET` to the name of the bucket.
3. Run the application and visit the dev page to view the list of schemas.
4. Complete a survey and verify it submits ok.
### Who can test

Anyone but @dhilton

### CODE REVIEW

File: app/schema_loader/schema_loader.py [hunk 1]
```diff
@@ -2,10 +2,17 @@
...
+import botocore
 
 
+class SchemaNotFound(Exception):
+    pass
+
 logger = logging.getLogger(__name__)
 
+# s3 = boto3.resource('s3')
+
```
Comment (warrenbailey) [inline app/schema_loader/schema_loader.py]: Can we remove the commented out code?

File: app/settings.py [hunk 2]
```diff
@@ -49,6 +49,8 @@ def get_key(key_name):
 EQ_SESSION_TIMEOUT = int(os.getenv('EQ_SESSION_TIMEOUT', '28800'))
 EQ_SECRET_KEY = os.getenv('EQ_SECRET_KEY', os.urandom(24))
 EQ_UA_ID = os.getenv('EQ_UA_ID', '')
+EQ_SCHEMA_BUCKET = os.getenv('EQ_SCHEMA_BUCKET', '')
+EQ_SCHEMA_REGISTRY_URL = os.getenv('EQ_SCHEMA_REGISTRY_URL', '')
```
Comment (warrenbailey) [inline app/settings.py]: This last env variable doesn't appear to be used anywhere? Is it needed?

Comment (dhilton) [inline app/settings.py]: Nope - I'll remove it.

**Comment by warrenbailey** [comment]
This works really well, just a few minor things and it's ready to go.

**Comment by warrenbailey** [comment]
Rebasing and making the code review comments

**Comment by warrenbailey** [comment]
LGTM 👍

**Pull request merged**